### PR TITLE
Fixes #9474: display confirmation when applying errata to CH, BZ1160740.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/9474
https://bugzilla.redhat.com/show_bug.cgi?id=1160740

Requires https://github.com/Katello/bastion/pull/45

### CODE REVIEW

**Comment by ehelms** [comment]
ACK

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html [hunk 1]
```diff
@@ -33,12 +33,18 @@
   </span>
 
   <span data-block="actions" bst-feature-flag="remote_actions">
+
+        <div bst-modal="applySelected()" model="contentHost"
```
Comment (ehelms) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html]: Indentation is off.

**Comment by waldenraines** [comment]
@ehelms updated.

**Comment by ehelms** [comment]
ACK

**Pull request merged**